examples/psp-power/src/main.rs

```diff
@@ -0,0 +1,17 @@
+#![no_std]
+#![no_main]
+
+psp::module!("sample_module", 1, 1);
+
+fn psp_main() {
+    psp::enable_home_button();
+    
+    unsafe{
+        psp::sys::power::sce_power_set_clock_frequency(333, 333, 166);    
+        
+        psp::dprint!("Hello PSP from rust!\n");
```

Comment:
You can use `psp::dprintln!` here instead of `psp::dprint!("...\n")`, the `println!` macro adds a newline automatically

Reply:
Changes have been made